src/components/Article.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from 'react-router-dom'
+import {useEffect, useState} from 'react'
+import axios from 'axios'
+
+function Article({articles}){
+
+    const [article, setArticle] = useState([])
```

Comment:
Would a single article be an array? While you'll be replacing this, the default value in state is representative of what is going to be used